spec/controllers/ops_controller/settings/common_spec.rb

```diff
@@ -231,26 +231,6 @@
       end
     end
 
-    context "#restore_password" do
```

Comment:
The test was replication specific, but I think the functionality was not.  Perhaps just change this to a different key to keep the test if it was used for other stuff?